src/components/Questions/Body.vue

```diff
@@ -8,6 +8,14 @@
       :hasQuizEnded="hasQuizEnded"
       :questionSetStates="questionSetStates"
       :currentQuestionIndex="currentQuestionIndex"
+      :title="title"
+      :subject="subject"
+      :testFormat="testFormat"
+      :maxMarks="maxMarks"
+      :numQuestions="numQuestions"
+      :quiz-time-limit="quizTimeLimit"
+      :questionSets = "questionSets"
```

Comment:
why send `questionSets` again if questionSetStates are available? 

if needed, update `questionSetStates` in Player.vue.. try to send as less information as possible across components. Else the app gets really slow.

Reply:
Yes replaced questionSets to questionSetStates from the instructionPage component.